doc/rados/operations/stretch-mode.rst

```diff
@@ -168,28 +168,33 @@ possible, if needed).
 
 .. _Changing Monitor elections: ../change-mon-elections
 
-Stretch Mode Limitations
-========================
-As implied by the setup, stretch mode only handles 2 sites with OSDs.
-
-While it is not enforced, you should run 2 monitors in each site plus
-a tiebreaker, for a total of 5. This is because OSDs can only connect
-to monitors in their own site when in stretch mode.
-
-You cannot use erasure coded pools with stretch mode. If you try, it will
-refuse, and it will not allow you to create EC pools once in stretch mode.
-
-You must create your own CRUSH rule which provides 2 copies in each site, and
-you must use 4 total copies with 2 in each site. If you have existing pools
-with non-default size/min_size, Ceph will object when you attempt to
-enable stretch mode.
-
-Because it runs with ``min_size 1`` when degraded, you should only use stretch
-mode with all-flash OSDs.  This minimizes the time needed to recover once
-connectivity is restored, and thus minimizes the potential for data loss.
-
-Hopefully, future development will extend this feature to support EC pools and
-running with more than 2 full sites.
+Limitations of Stretch Mode 
+===========================
+When using stretch mode, OSDs must be located at exactly two sites. 
+
+Two monitors should be run in each data center, plus a tiebreaker in a third
+(or in the cloud) for a total of five monitors. While in stretch mode, OSDs
+will connect only to monitors within the data center in which they are located.
+OSDs *DO NOT* connect to the tiebreaker monitor.
```

Comment:
Do we really need this sentence? The previous one seems very clear.

Reply:
@gregsfortytwo, I've been working with this text for the past week, so I don't have fresh eyes. That means that I won't be able to give you the kind of answer that I could if I were reading this for the first time.

I think that it's better if this sentence is left in the text. I know that it probably seems completely obvious to you that OSDs don't connect to the tiebreaker monitor, but I suspect that it seems obvious to you because in the matter of stretch mode, this is the world you created and the rest of us are just tourists in it.